src/Services/Identity/Query/Infrastructure.EventBus/DependencyInjection/Extensions/RabbitMqBusFactoryConfiguratorExtensions.cs

```diff
@@ -9,9 +9,9 @@ internal static class RabbitMqBusFactoryConfiguratorExtensions
 {
     public static void ConfigureEventReceiveEndpoints(this IRabbitMqBusFactoryConfigurator cfg, IRegistrationContext context)
     {
-        cfg.ConfigureEventReceiveEndpoint<UserDeletedConsumer, DomainEvent.UserDeleted>(context);
-        cfg.ConfigureEventReceiveEndpoint<UserRegisteredConsumer, DomainEvent.UserRegistered>(context);
-        cfg.ConfigureEventReceiveEndpoint<UserPasswordChangedConsumer, DomainEvent.PasswordChanged>(context);
+        cfg.ConfigureEventReceiveEndpoint<ProjectUserDetailsWhenUserChangedConsumer, DomainEvent.UserDeleted>(context);
+        cfg.ConfigureEventReceiveEndpoint<ProjectUserDetailsWhenUserChangedConsumer, DomainEvent.UserRegistered>(context);
+        cfg.ConfigureEventReceiveEndpoint<ProjectUserDetailsWhenUserChangedConsumer, DomainEvent.UserPasswordChanged>(context);
     }
 
     private static void ConfigureEventReceiveEndpoint<TConsumer, TEvent>(this IRabbitMqBusFactoryConfigurator bus, IRegistrationContext context)
```

Comment:
Please, adjust for:

` queueName: $"identity.query-stack.{typeof(TConsumer).ToKebabCaseString()}.{typeof(TEvent).ToKebabCaseString()}",`